make-release.sh

```diff
@@ -107,24 +99,15 @@ else
 fi
 set -e
 
-# change VERSION file
-echo "${VERSION}" > VERSION
-
 QUAY_REPO="quay.io/devfile/devworkspace-controller:${VERSION}"
 docker build -t "${QUAY_REPO}" -f ./build/Dockerfile .
 docker push "${QUAY_REPO}"
 
 set -x
 bash -x ./deploy/generate-deployment.sh --use-defaults --default-image ${QUAY_REPO}
 
-# tag the release if the VERSION file has changed
```

Comment:
Please do not remove this section, as generate-deployment.sh will generate templates, that we will need to commit

Reply:
oh, I thought this was only about hte VERSION file